dragen-covid-pipeline-epitrax_v3LW.R

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env Rscript
+# Author: OLP/01/2022
+# Usage: Getting metadata
+# Parameters: Only one: Run Name i.e. Rscript / . . . ./dragen-covid-pipeline-epitrax-v2.R UT-A01290-220113
+
+library("dplyr", warn.conflicts = FALSE)
+library("grid")
+library("rbin")#
+library("tidyverse")
+library("lubridate", warn.conflicts = FALSE)
+library("tibble")
+
+# Step 1: date from argument
+#+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+args = commandArgs(trailingOnly=TRUE)
+#args[1]<-c('UT-A01290-220304'); # only to works on script preparation
+s<- paste('/Volumes/NGS/Analysis/covidseq/',args[1],sep="/"); head(s)
+ss<-gsub("\\.*.*/","", s);date<-ymd(substr(ss, 11, 16)); head(ss); head(date)
+
+# Step 2: Reading covidseq_summary
+#+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+loc<-paste(paste('/Volumes/NGS/Analysis/covidseq',args[1],sep="/"),"sumcovidseq",sep ="/"); head(loc)
+covidseq_summary<-read.csv(paste(loc,"covidseq_summary.csv", sep="/"));
+covidseq_summary<-covidseq_summary[,c(1,9)]; colnames(covidseq_summary)[2] <- 'Test result';#  sample and pangolin_lineage
+
+
+# Step 3: Selecting only PatientSamples from the SampleSheet
+#+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+# Empy weels. 
+# The SampleSheet was modified on 2022-02-26 where Empty well is callep as Empty but if this
+# script is used for data before that the script filter the Empty well in the following lines
+sample<-covidseq_summary$sample; sample<-gsub("\\Emp.*","", sample);
+covidseq_summary$sample<-sample
+
+loc2 <-paste("/Volumes/NGS/Analysis/covidseq",args[1],sep="/"); head(loc2)
+SampleSheet<-read.csv(paste(loc2,"covidseq_output/SampleSheet_Intermediate.csv", sep="/"));
+ss1<-tail(SampleSheet,-13);#ss : SampleSheet
+ss<-ss1[,c(1,3)]; # selecting sampleNumber and PatientSample
+colnames(ss) <- c("Sample_Accession","Sample_Type");
+ss<-filter(ss, Sample_Type == "PatientSample"); 
+# inner join
+covidseq_summary_filtered<-merge(ss,covidseq_summary, by.x="Sample_Accession", by.y= "sample")
+covidseq_summary_filtered<-covidseq_summary_filtered[,c(1,3)]
+head(covidseq_summary_filtered)
+
+#Step 4: Detecting last files from NAS: NGS_Covid_...*csv & All_ncovid_NoNGS...*csv files
+#++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+tmpshot <- fileSnapshot("/Volumes/IDGenomics_NAS/NextStrain/EpiTrax/", pattern=glob2rx("All*.csv"))#
```

Comment:
This line has a hashtag at the end of it. Is there a comment that got left out?

Reply:
Thanks, I will delete the hashtag at the end of the line